src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -193,6 +194,15 @@ def from_dict(cls, r_dict):
         return Role(r_dict['name'], r_dict['description'],
                     r_dict['scopes_permissions'])
 
+    @classmethod
+    def map_to_system_roles(cls, role):
+        matches = []
+        for rn in SYSTEM_ROLES_NAMES:
+            closest_match = difflib.get_close_matches(role.lower(), SYSTEM_ROLES_NAMES[rn], n=1, cutoff=0.8)
```

Comment:
That's a clear no. In [Security](https://cheatsheetseries.owasp.org/cheatsheets/Secure_Product_Design_Cheat_Sheet.html), you don't play with fuzzy matches. Either authorization scopes match 100% or they don't.

What's the sense of allowing `damin` role to be granted `admin` rights?

Reply:
That make sense.
 The `get_close_matches` cutoff parameter was set to `0.8` (pretty strict) to forbid things like `damin` and allow `admin` even though the real role name is `administator`

But the security risk is clear. I have removed it